src/codeocean/computation.py

```diff
@@ -141,17 +141,18 @@ def run_capsule(self, run_params: RunParams) -> Computation:
 
         return Computation.from_dict(res.json())
 
-    def wait_until_completed(self, computation: Computation) -> Computation:
+    def wait_until_completed(self, computation: Computation, polling_interval: int = 5) -> Computation:
         """
         Polls the given computation until it reaches the 'Completed' or 'Failed' state.
         """
+        assert polling_interval >= 5, "polling_interval should be greater than or equal to 5"
```

Comment:
Raising a `ValueError` is preferable over assertions. The latter is more for debugging.